webpack.config.js

```diff
@@ -2,6 +2,7 @@ const path = require('path');
 const TerserPlugin = require('terser-webpack-plugin');
 
 module.exports = {
+    mode: 'node',
```

Comment:
https://webpack.js.org/configuration/mode/

Should be `none`

Reply:
Thanks for noticing! Updated.